### ISSUE TITLE
osx-arm64 builds missing

### ISSUE THREAD

**Issue by bschindler** [conda-forge/kornia-rs-feedstock#14]
### Comment:

Hi - I would love to update kornia in our env but cannot because we are on mac and the kornia-rs builds are only available for mac x86, but not mac arm64. Could this arch be added or is this really hard?

**Comment by edgarriba** [comment]
what python version are you targetting ? In principle we support arm64. See: https://github.com/kornia/kornia-rs/blob/main/.github/workflows/python_release.yml

**Comment by bschindler** [comment]
We are targeting 3.10, 3.11 soon hopefully. So in the end, this is a simple manner of enabling that architecture and triggering the bot?

**Comment by mukhery** [comment]
Yeah, maybe we just need to follow https://conda-forge.org/blog/2020/10/29/macos-arm64/. @bschindler is there another rust-based conda-forge package you know of that has these builds? That would also help us make sure we're enabling it correctly.

**Comment by bschindler** [comment]
I am really more of a conda user than a developer, but I know that ruff has a rust core and is installable on arm64. I guess its possible to peek there...?

**Comment by mukhery** [comment]
Thanks, I'll take a look there when I get some free time

**Comment by mukhery** [comment]
Resolved with #16
